AGENTS.md

```diff
@@ -7,13 +7,12 @@
 - **Type Check**: `pnpm check-types`
 - **Test All**: `pnpm test` / `pnpm test:watch`
 - **Single Test**: `vitest run <path/to/test.ts>`
-- **Format**: `pnpm format` / `pnpm format:check`
 
 ## Code Style
 
 - **TypeScript**: Strict mode, consistent type definitions/imports
-- **Formatting**: Prettier (single quotes, 120 width, no semicolons, trailing commas)
-- **Linting**: ESLint with perfectionist (alphabetical imports/sorting)
+- **Formatting**: Biome (single quotes, 120 width, no semicolons, trailing commas)
+- **Linting**: Biome with perfectionist (alphabetical imports/sorting)
```

Comment:
I think we no longer use `perfectionist`? It's also mentioned in the **Imports** section 4 lines down.

Reply:
We do use the equivalent in biome called `organizeImports`. I'll perfectionist from here